src/components/charts/DataTable.vue

```diff
@@ -14,7 +14,7 @@
       <td
         v-for="y in years" :key="y"
         :style="{'background-color': d[y].color}"
-        class="text-center"
+        :class="{ 'invert-text' : d[y].color === '#00A03C' || d[y].color === '#00A0E6' }"
```

Comment:
I suggest extending the colorcoding service with an additional method isTextInverted(color) or something like that, instead of using these hardcoded values

Reply:
done